tatoeba2/models.py

```diff
@@ -319,6 +332,10 @@ class Users(models.Model):
     homepage = models.CharField(max_length=255)
     image = models.CharField(max_length=255)
     country_id = models.CharField(max_length=2, blank=True)
+    lang = models.CharField(max_length=100, blank=True)
+    is_public = models.IntegerField()
```

Comment:
`lang` and `is_public` are no longer columns in the `users` table. They have been removed when `settings` was introduced: https://github.com/Tatoeba/tatoeba2/commit/b51fb6809b94741b0e4777879141756ca4b5682a#diff-dad9a1167675fc8ae00ea7942b68c97a